subsys/bluetooth/controller/ll_sw/ull_adv_sync.c

```diff
@@ -1676,10 +1678,11 @@ static inline void sync_info_offset_fill(struct pdu_adv_sync_info *si,
 	offs = HAL_TICKER_TICKS_TO_US(ticks_offset) - start_us;
 	offs = offs / OFFS_UNIT_30_US;
 	if (!!(offs >> OFFS_UNIT_BITS)) {
-		si->offs = offs / (OFFS_UNIT_300_US / OFFS_UNIT_30_US);
+		si->offs = sys_cpu_to_le16(offs / (OFFS_UNIT_300_US /
+						   OFFS_UNIT_30_US));
```

Comment:
The `offs / (OFFS_UNIT_300_US / OFFS_UNIT_30_US)` calculation seems to be done more than 1 place; consider a common function to calculate it.

Reply:
Will send a new PR to refactor this.